demo-app/src/POCMap.jsx

```diff
@@ -3,14 +3,66 @@ import { Box } from 'grommet';
 import { Grommet } from 'grommet-icons';
 import {
   Controls,
+  Cluster,
   Map,
   Marker,
   MarkerCluster,
   Pin,
 } from 'grommet-leaflet-core';
-
+import {
+  StatusCriticalSmall,
+  StatusGoodSmall,
+  StatusWarningSmall,
+  StatusUnknownSmall,
+} from 'grommet-icons';
 import { generateLocations, userLocation } from './utils/locations';
 import { ClusterPopup } from './ClusterPopup';
+import { getClusterStatus, getClusterSize } from './utils/status';
+
+const markerKinds = {
+  default: {
```

Comment:
Related question earlier, do we want to embrace this pattern with `default` or follow Grommet precedent?

Reply:
Link to reply above: https://github.com/grommet/grommet-leaflet/pull/28#discussion_r1276816731